sysbox-in-docker/Makefile

```diff
@@ -0,0 +1,90 @@
+#
+# Sysbox-in-docker Makefile.
+#
+# The targets in this Makefile offer a simple approach to build a docker-image
+# to run a containerized Sysbox environment.
+#
+
+
+SHELL:=/bin/bash
+SIND_IMAGE := nestybox/sysbox-in-docker
+
+
+.DEFAULT := help
+
+.PHONY: help            \
+	centos-8        \
+	debian-buster   \
+	debian-bullseye \
+	fedora-31       \
+	fedora-32       \
+	ubuntu-bionic   \
+	ubuntu-focal
+
+
+.DEFAULT := help
+
+help: ## Show supported docker images
+	@awk 'BEGIN {FS = ":.*##"; printf "\n\033[1mUsage:\n  \
+	make sysbox-in-docker \033[36m<distro-release>\033[0m\n\n"} \
+	/^[a-zA-Z0-9_-]+:.*?##/ { printf "  \033[36m%-25s\033[0m %s\n", $$1, $$2 } /^##@/ \
+	{ printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
+
+
+LAUNCH_INSTRUCTION := docker run -it --privileged --rm --hostname sysbox-in-docker --name sysbox-in-docker \
+                -v /var/tmp/sysbox-test-var-lib-docker:/var/lib/docker \
```

Comment:
Use other volumes (not the sysbox-test-var-lib-docker volume which is reserved for the sysbox test env).

Reply:
Done.